zomes/chat/src/entries/message/handlers.rs

```diff
@@ -209,6 +209,33 @@ fn active_chatters(chatters_path: Path) -> ChatResult<(usize, Vec<AgentPubKey>)>
     Ok((total, active))
 }
 
+pub(crate) fn get_active_chatters() -> ChatResult<ActiveChatters> {
+    let me = agent_info()?.agent_latest_pubkey;
+    let chatters_path: Path = chatters_path();
+    let (_total, mut chatters) = active_chatters(chatters_path)?;
+    chatters.retain(|a| *a != me);
+    Ok(ActiveChatters { chatters })
+}
+
+pub(crate) fn signal_specific_chatters(input: SignalSpecificInput) -> ChatResult<()> {
+    let mut chatters = input.chatters;
+
+    if let Some(include_active_chatters) = input.include_active_chatters {
+      if include_active_chatters {
+        let active_chatters_result = get_active_chatters();
+        if let Ok(mut active_chatters) = active_chatters_result {
```

Comment:
Not sure if I should be passing the possible error from this call up as the result of `signal_specific_chatters`, but there's still utility even if `get_active_chatters` fails.